src/main/java/edu/ucla/library/iiif/fester/CsvParser.java

```diff
@@ -348,11 +350,13 @@ private void extractPageMetadata(final String... aRow) throws CsvParsingExceptio
         final Optional<Float> mediaDuration =
                 (Optional<Float>) getMetadata(aRow, myCsvHeaders.getMediaDurationIndex(), Float.class, aPath);
         final Optional<String> mediaFormat = getMetadata(aRow, myCsvHeaders.getMediaFormatIndex());
-        final Optional<String> audioVideoAccessUrl = getMetadata(aRow, myCsvHeaders.getAudioVideoAccessUrlIndex());
+        final Optional<String> audioVideoAccessUrl = getMetadata(aRow, myCsvHeaders.getContentAccessUrlIndex());
 
         if (Constants.IIIF_API_V2.equals(aIiifVersion)) {
+            //if (hasAVContent(mediaWidth, mediaHeight, mediaDuration, mediaFormat, audioVideoAccessUrl)) {
             if (mediaWidth.isPresent() || mediaHeight.isPresent() || mediaDuration.isPresent() ||
-                    mediaFormat.isPresent() || audioVideoAccessUrl.isPresent()) {
+                    mediaFormat.isPresent() || (audioVideoAccessUrl.isPresent() &&
+                    audioVideoAccessUrl.toString().contains(AV_URL_STRING))) {
                 throw new CsvParsingException(MessageCodes.MFS_168, rowId, aPath);
             }
         } else { // Constants.IIIF_API_V3
```

Comment:
In the case of v3, might we also want to check if the access URL _doesn't_ contain `AV_URL_STRING` (and throw the exception if so) in the switch statement below?